client/src/components/common/modal/Modal.js

```diff
@@ -6,6 +6,7 @@ function Modal({
 	modalOpen,
 	children,
 	setModal,
+	handleOkayClick,
```

Comment:
This function can be appropriately named as `handleConfirm` instead of `handleOkayClick`. I find the suggested name more descriptive in my opinion